src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -271,6 +272,7 @@
   </ItemGroup>
 
   <Target Name="ILRepacker"
+      Condition=" '$(_ILRepackEnabled)' == 'true' "
```

Comment:
It looks like we only have 3 assemblies left?
https://github.com/xamarin/xamarin-android/blob/e987ac458536e59a8329a06d5c5d5f4d4ea2c6b6/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets#L268-L270

Can we remove these and not pack them?

We could also consider `System.Text.Json` over `Newtonsoft`.

Reply:
Moving from Newtonsoft might help. As the others I'm not sure. 
We should do that in a new PR